src/De/DeBson.jl

```diff
@@ -0,0 +1,52 @@
+module DeBson
+
+# Ensure BSON package is available
+using BSON
+
+export deser_bson
+
+"""
+    deser_bson(::Type{T}, bson_path::String; kw...) -> T
+
+Deserialize a BSON file specified by `bson_path` into an instance of type `T`.
+
+# Arguments
+- `T`: The type to deserialize into.
+- `bson_path`: Path to the BSON file.
+
+# Keyword Arguments
+- `kw`: Additional keyword arguments for customization.
+
+# Examples
+```julia
+struct Record
+    count::Float64
+end
+
+struct Data
+    id::Int64
+    name::String
+    body::Record
+end
+
+# Assuming `data.bson` represents a `Data` instance
+deserialized_data = deser_bson(Data, "data.bson")
+"""
+function deser_bson(::Type{T}, bson_path::String; kw...) where T
```

Comment:
Other functions in Serde require byte array or string as input. Could you do something similar here?